packages/management/addons-manager/bundle/config/values.yaml

```diff
@@ -10,6 +10,7 @@ tanzuAddonsManager:
     priorityClassName: null
     nodeSelector: null
     tolerations: []
+    webhookServerPort: 9865
```

Comment:
We also need to make the same changes in framework and tkg composite package.

Reply:
added these data values in framework and tkg composite package as well